Minor update to getting started steps (#527)
serverless-nextjs/serverless-next.js: README.md

### Getting started

Install the component:

```

> npm i @sls-next/serverless-component -D

Reviewer: You don't actually need to `npm install` the component. I've deliberately left this step out :)

Author: Thanks @danielcondemarin.
Can you tell me why I've got the error (see description screenshot) when running serverless deploy?

Reviewer: That version is genuinely not in the registry:

![image](https://user-images.githubusercontent.com/1122442/89105815-66f92c00-d41c-11ea-8aed-73c95ad72bc3.png)

Author: oops :)
Have no idea where did I take that version from. Probably my Friday imagination :)

Is it true that `component: serverless-next.js` should be replaced with `component: "@sls-next/serverless-component@{version_here}"` across Readme file ?

Reviewer: Yes that's true. serverless-next.js used to be the old name of the package before I created the org @sls-next :)

```

Add your next application to the serverless.yml:

```yml
# serverless.yml

myNextApplication:

>   component: "@sls-next/serverless-component@{version_here}" # it is recommended you pin the latest stable version of serverless-next.js
>   # replace {version_here} with the one from package.json

Reviewer: ```suggestion
  # it is recommended you pin the latest stable version of serverless-next.js
  # replace {version_here} with a stable npm version available
  component: "@sls-next/serverless-component@{version_here}" 
```
